src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -32,7 +32,7 @@
     public static final String VALID_FILE_ADDRESS_CS2101 = "./src/test/java/seedu/address/testutil/cs2101.bat";
     public static final String VALID_MAC_FILE_ADDRESS_CS2101 = "./src/test/java/seedu/address/testutil/cs2101.sh";
     public static final String VALID_MAC_FILE_ADDRESS_TESTFILE = "./src/test/java/seedu/address/testutil/testFile.sh";
-    public static final String VALID_LABEL = "testLabel";
+    public static final String VALID_LABEL = "validLabel";
```

Comment:
Perhaps don't change this since it won't be consistent with TypicalTags?

Reply:
This conversation is discussed above.